README.md

```diff
@@ -2,11 +2,37 @@
 
 A Survey of:
 
-_Does the first programming language someone learns influence whether they prefer R or Python (or have no preference)?_
+_Does the first programming language learned influence whether someone prefersPython or R when it comes to data analysis._
 
 | Team Members | Github Handle |
 |--------------|---------------|
 | Brenden Everitt | everittB |
 | Miliban Keyim | mkeyim |
 | Maninder Kohli | ksm45 |
-| Phuntsok Tseten | phuntsoktseten |  
+| Phuntsok Tseten | phuntsoktseten |
+
+
+#### Chosen Questions:
+* Which language do you prefer for data analysis:
+- Python or R. This is the response question.
+* What is the first computing language you learned?
+  -  This survey question explores the influence of specific languages for the preference of Python or R.
+
+* What was your undergraduate degree in? 
+  - Educational background will influence what programming language one prefers to learn first.
+
+* Are your analysis projects typically small(10K rows), medium(10-1M Rows), big (1m+ rows) ?
+  - The size of a data analysis project might influence the language a person to use because of difference in speed and ease of setup/use.
```

Comment:
```suggestion
  - The size of a data analysis project might influence the language a person tends to use because of difference in speed and ease of setup/use.
```